### PR TITLE
type-connect part 8: teams, wallets

### PR DESCRIPTION
**Pull request by chrisnojima**

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

File: shared/wallets/confirm-form/container.js [hunk 1]
```diff
@@ -8,4 +8,5 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({})
 
 const mergeProps = (stateProps, dispatchProps) => ({})
 
+// $FlowIssue TODO
```
Comment (akalin-keybase) [inline shared/wallets/confirm-form/container.js]: what's the issue here?

File: shared/wallets/send-form/footer/container.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({})
 
 const mergeProps = (stateProps, dispatchProps) => ({})
 
+// $FlowIssue we're not sending anything
```
Comment (akalin-keybase) [inline shared/wallets/send-form/footer/container.js]: and here?

File: shared/wallets/confirm-form/container.js (hunk 1 above)
Comment (chrisnojima) [reply shared/wallets/confirm-form/container.js]: these weren't implemented yet so we werent passing any props which is incorrect. cjbs pr fills this stuff in i believe

File: shared/wallets/send-form/footer/container.js (hunk 2 above)
Comment (chrisnojima) [reply shared/wallets/send-form/footer/container.js]: same, these are all empty which is wrong

**Pull request merged**